libs/designer/src/lib/ui/settings/sections/general.tsx

```diff
@@ -104,12 +107,22 @@ export const General = ({
     defaultMessage: 'Array',
     description: 'title for array dropdown input setting',
   });
+  const invokerConnectionTitle = intl.formatMessage({
+    defaultMessage:'Use Invoker\'s Connection',
```

Comment:
```suggestion
    defaultMessage:"Use Invoker's Connection",
```

always prefer double quotes if the string includes a single quote. This helps avoid errors/bugs related to escapes.